servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -574,13 +574,13 @@ private TempBlockMeta createBlockMetaInternal(long sessionId, long blockId,
         mMetaManager.addTempBlockMeta(tempBlock);
       } catch (WorkerOutOfSpaceException ose) {
         // If we reach here, allocator is not working properly
-        LOG.error("Unexpected failure: " + initialBlockSize + " bytes allocated at " + location
-            + " by allocator, but addTempBlockMeta failed");
+        LOG.error("Unexpected failure: {} bytes allocated at {} by allocator, "
+            + "but addTempBlockMeta failed", initialBlockSize, location);
         throw Throwables.propagate(ose);
       } catch (BlockAlreadyExistsException aee) {
         // If we reach here, allocator is not working properly
-        LOG.error("Unexpected failure: " + initialBlockSize + " bytes allocated at " + location
-            + " by allocator, but addTempBlockMeta failed");
+        LOG.error("Unexpected failure: %d bytes allocated at {} by allocator, "
```

Comment:
is it correct to have `%d` here? I thought slf4j doesn't know `%d` but only `{}`?